src/main/java/com/cloudbees/plugins/credentials/impl/BaseStandardCredentials.java

```diff
@@ -221,12 +221,10 @@ public final FormValidation doCheckId(@ContextInPath ModelObject context, @Query
             }
             if (!(context instanceof Jenkins)) {
                 // CredentialsProvider.lookupStores(User) does not return SystemCredentialsProvider.
-                Jenkins j = Jenkins.getInstance();
-                if (j != null) {
-                    problem = checkForDuplicates(value, context, j);
-                    if (problem != null) {
-                        return problem;
-                    }
+                Jenkins j = Jenkins.get();
```

Comment:
This is technically a change in behavior. Before, it would be a no-op if it returns null. Now this will throw an exception.

Reply:
Exactly. The method `doCheckId` is about form validation. Therefore it should run on Jenkins master and Jenkins should be up and running.

Reply:
Ah, then there really was no point to not throwing an exception initially. Good catch!